src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -64,6 +64,11 @@
    (str "/#/administration/catalogue-items/" catalogue-item-id)
    (text :t.administration/view)])
 
+(defn- to-edit-catalogue-item [catalogue-item-id]
```

Comment:
could this be shared with `catalogue-item/edit-button`?

Reply:
Almost... It seems that one is btn-primary whereas the other is btn-secondary. It seems slightly messier to pass that information to the function than to have two separate functions.

Also, it would build a dependence between rems.administration.catalogue-items and .catalogue-item where there is currently none (except via page change).

Therefore, it would seem a tiny bit better to keep them separate for now.

Reply:
fair enough